modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheClusterMetricsMXBeanImpl.java

```diff
@@ -17,515 +17,121 @@
 
 package org.apache.ignite.internal.processors.cache;
 
-import java.util.Collections;
+import javax.cache.management.CacheMXBean;
+import javax.cache.management.CacheStatisticsMXBean;
 import org.apache.ignite.IgniteCache;
-import org.apache.ignite.mxbean.CacheMetricsMXBean;
 
 /**
- * Management bean that provides access to {@link IgniteCache IgniteCache}.
+ * MX bean that keeps support of JCache specification.
  */
-public class CacheClusterMetricsMXBeanImpl implements CacheMetricsMXBean {
+public class CacheClusterMetricsMXBeanImpl implements CacheStatisticsMXBean, CacheMXBean {
```

Comment:
It this bean still provide cluster-wide values?

Reply:
Yes. It uses `IgniteCache#metrics()`. There is also `IgniteCache#localMetrics()`